getting-started/templates/systemlink-values.yaml

```diff
@@ -181,7 +181,7 @@ minio:
     hostname: "systemlink-minio-api.example.com"
   ## Comma, semi-colon or space separated list of buckets to create at initialization (only in standalone mode).
   ##
-  defaultBuckets: "systemlink-file-ingestion;systemlink-dataframe"
+  defaultBuckets: "systemlink-file-ingestion;systemlink-dataframe;systemlink-notebook-execution"
```

Comment:
This should be recorded as a breaking change in the breaking change document. Although GM does not use minio so this might not matter for them.